components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -87,18 +87,72 @@ abstract class EngineSession(
     ) {
         companion object {
             internal const val NONE: Int = 0
+            /**
+             * Blocks advertisement trackers.
+             */
             const val AD: Int = 1 shl 1
+            /**
+             * Blocks analytics trackers.
+             */
             const val ANALYTICS: Int = 1 shl 2
+            /**
+             * Blocks social trackers.
+             */
             const val SOCIAL: Int = 1 shl 3
+            /**
+             * Blocks content trackers.
+             * May cause issues with some web sites.
+             */
             const val CONTENT: Int = 1 shl 4
             // This policy is just to align categories with GeckoView (which has AT_TEST = 1 << 5)
             const val TEST: Int = 1 shl 5
+            /**
+             * Blocks cryptocurrency miners.
+             */
             const val CRYPTOMINING = 1 shl 6
+            /**
+             * Blocks fingerprinting trackers.
+             */
             const val FINGERPRINTING = 1 shl 7
-            internal const val ALL: Int = AD + ANALYTICS + SOCIAL + CONTENT + TEST + CRYPTOMINING + FINGERPRINTING
+            /**
+             * Blocks malware sites.
+             */
+            const val SAFE_BROWSING_MALWARE = 1 shl 10
+            /**
+             * Blocks unwanted sites.
+             */
+            const val SAFE_BROWSING_UNWANTED = 1 shl 11
+            /**
+             * Blocks harmful sites.
+             */
+            const val SAFE_BROWSING_HARMFUL = 1 shl 12
+            /**
+             * Blocks phishing sites.
+             */
+            const val SAFE_BROWSING_PHISHING = 1 shl 13
+            /**
+             * Blocks all unsafe sites.
+             */
+            const val SAFE_BROWSING_ALL =
+                SAFE_BROWSING_MALWARE + SAFE_BROWSING_UNWANTED + SAFE_BROWSING_HARMFUL + SAFE_BROWSING_PHISHING
+
+            internal const val RECOMMENDED: Int = AD + ANALYTICS + SOCIAL + TEST + SAFE_BROWSING_ALL
+
+            internal const val ALL: Int = RECOMMENDED + CRYPTOMINING + FINGERPRINTING + CONTENT
 
             fun none() = TrackingProtectionPolicy(NONE)
+
+            /**
+             * Strict content blocking categories.
+             * A combination of the [recommended] categories plus [CRYPTOMINING], [FINGERPRINTING] and [CONTENT].
+             * May cause issues with some web sites.
+             */
             fun all() = TrackingProtectionPolicyForSessionTypes(ALL)
+            /**
+             * Recommended content blocking categories.
```

Comment:
nit: see above would prefer:

`
Combining the [AD], [ANALYTICS], [SOCIAL], [TEST] categories plus [SAFE_BROWSING_ALL]. This is the recommended setting.
`